app/templates/components/article-featured-video.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="video-container{{unless isPlayerLoading ' player-ready'}}{{if isPlayed ' is-played'}}"
```

Comment:
Is there a reason to have this additional wrapper? If not, then we could have these classes and style defined as computed properties in the component.